atmat/attests/pytests.m

```diff
@@ -268,8 +268,8 @@ function checkattr(rm, rp)
                 testCase.verifyEqual(rm{end}.Alphay, rp{3}.Alphay, AbsTol=1.e-10);
                 testCase.verifyEqual(rm{end}.Betax, rp{3}.Betax, RelTol=1.e-10);
                 testCase.verifyEqual(rm{end}.Betay, rp{3}.Betay, RelTol=1.e-10);
-                testCase.verifyEqual(rm{end}.Qpx, rp{3}.Qpx, RelTol=1.e-8);
-                testCase.verifyEqual(rm{end}.Qpy, rp{3}.Qpy, RelTol=1.e-8);
+                testCase.verifyEqual(rm{end}.chromx_arr, rp{3}.chromx_arr, RelTol=1.e-8);
+                testCase.verifyEqual(rm{end}.chromy_arr, rp{3}.chromy_arr, RelTol=1.e-8);
```

Comment:
I think it's missing a conversion from numpy array to double:

```Matlab
testCase.verifyEqual(rm{end}.chromx_arr, double(rp{3}.chromx_arr), RelTol=1.e-8);
testCase.verifyEqual(rm{end}.chromy_arr, double(rp{3}.chromy_arr), RelTol=1.e-8);
```
Try this !

Reply:
Thank you!